package.json

```diff
@@ -22,7 +22,7 @@
     "build-storybook": "build-storybook -c storybook/.storybook",
     "test": "npm run test:unit",
     "test:unit": "jest",
-    "test:acceptance": "testcafe chrome,firefox test/acceptance/ --app \"npm start\"",
+    "test:acceptance": "testcafe chrome,firefox test/acceptance/",
```

Comment:
this change doesn't seem to work for me. does the app need to be started before we run acceptance tests? because without this, nothing will be running on port 3000 when acceptance tests are executed